charts/trustify-infrastructure/Chart.lock

```diff
@@ -1,12 +0,0 @@
-dependencies:
```

Comment:
Why do we need to remove the lock file?

Reply:
Good question.
* I sent a PR and CI-lint complained about missing lock file
  * I added the lock file
    * CI-lint complained about `helm repo add otel`
      * I added that
        * CI-lint goes crazy saying that repo does't exist
          * Removed the lock file and CI pass :heavy_check_mark:

Reply:
```
helm version --short
v3.16.4+g7877b45
```

Reply:
Repo added without lock file -- CI-lint works https://github.com/trustification/trustify-helm-charts/pull/15/files

Reply:
I'll downgrade my helm version as I saw this note here: https://github.com/Azure/setup-helm?tab=readme-ov-file#example, and then I'll add the lock file back and see what happens

Reply:
not related to helm version...
```
helm version --short
v3.13.3+gc8b9489
```